actionpack/CHANGELOG.md

```diff
@@ -2,4 +2,20 @@
 
     *Hartley McGuire*
 
+*   Use the correct path when the resource is singular.
+
+    Before:
+    ```Ruby
```

Comment:
```suggestion
    ```ruby
```

Reply:
I'll go ahead and change this but this is a bit of a subjective suggestion here, especially since markdown isn't case sensitive for Language designation and the end user wouldn't actually see this rendered out.

Reply:
Yes it's not case sensitive but we need to consistent across the codebase.